node/consensus/src/lib.rs

```diff
@@ -74,6 +75,8 @@ pub struct Consensus<N: Network> {
     solutions_queue: Arc<Mutex<IndexMap<PuzzleCommitment<N>, ProverSolution<N>>>>,
     /// The unconfirmed transactions queue.
     transactions_queue: Arc<Mutex<BTreeMap<TransactionKey<N>, Transaction<N>>>>,
+    /// Size of transactions queue.
+    transactions_queue_size: Arc<Mutex<TransactionSize>>,
```

Comment:
this should be an `Arc<AtomicI64>`; it can be another type alias (e.g. `AtomicTransactionSize`) if practical

Reply:
753d6d0c5